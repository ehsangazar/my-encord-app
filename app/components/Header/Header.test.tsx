import renderer from "react-test-renderer";

import Header from "./Header";

it("Header component should render correctly", () => {
  const tree = renderer.create(<Header>Click me</Header>).toJSON();
  expect(tree).toMatchSnapshot();
});
