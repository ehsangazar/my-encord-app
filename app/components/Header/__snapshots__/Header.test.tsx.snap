// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
<header
  className="flex items-center justify-between p-4 bg-gray-800 border-b"
>
  <div
    className="max-w-4xl mx-auto w-full flex items-center justify-between"
  >
    <div
      className="flex items-center"
    >
      <h2
        className="text-xl font-semibold text-gray-800 dark:text-white"
      >
        My Encord App
      </h2>
    </div>
  </div>
</header>
`;
